connector/servicegraphconnector/config.go

```diff
@@ -0,0 +1,44 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package servicegraphconnector // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/servicegraphconnector"
+
+import (
+	"time"
+)
+
+// Config defines the configuration options for servicegraphconnector.
+type Config struct {
+
+	// MetricsExporter is the name of the metrics exporter to use to ship metrics.
+	MetricsExporter string `mapstructure:"metrics_exporter"`
```

Comment:
Do we still need this as a connector? Won't the connector get a metrics consumer on its TracesToMetrics constructor?

Reply:
We don't need this anymore. Good catch!